### ISSUE TITLE
Proposal: prohibit bindings with no value

### ISSUE THREAD

**Issue by yyx990803** [vuejs/eslint-plugin-vue#384]
Noticed some users get confused when they think they are binding an empty string with `v-bind`:

``` vue
<foo :bar=""/>
```

This actually produces incorrect render function code and cause webpack to report a hard-to-understand error:

```
error  in ./src/App.vue

Syntax Error: Unexpected token (1:98)


 @ ./src/App.vue 10:0-312
 @ ./src/main.js
 @ multi (webpack)-dev-server/client?http://localhost:8080/ (webpack)/hot/dev-server.js ./src/main.js
```

Would be great to consider this an error and include in the `essential` config.

**Comment by yyx990803** [comment]
On nvm ----- it already does ;)
